src/ert/scheduler/job.py

```diff
@@ -104,7 +104,7 @@ async def _send(self, state: State) -> None:
         event = CloudEvent(
             {
                 "type": _queue_state_event_type(status),
-                "source": f"/etc/ensemble/{self._scheduler._ens_id}/real/{self.iens}",
+                "source": f"/ert/ensemble/{self._scheduler._ens_id}/real/{self.iens}",
```

Comment:
No way did I really write `etc` and not notice??? :sweat_smile:

Reply:
Your finger muscle memory is forgiven